template/README.md

```diff
@@ -487,55 +487,91 @@ __About Promises:__ This project also includes a [Promise polyfill](https://gith
 
 You can make a GET request like this:
 ```javascript
-class MyComponent extends Component {
+import React, { Component, PropTypes } from 'react';
+
+const { string } = PropTypes;
+
+class RepoList extends Component {
+  static propTypes = {
```

Comment:
Let's remove propTypes. Not essential to this example and introduces class properties which we don't want for now.